vignettes/create-individual.Rmd

```diff
@@ -55,3 +55,30 @@ setParameterValuesByPath(parameterPaths = individual$distributedParameters$paths
                          values = individual$distributedParameters$values,
                          simulation = sim)
 ```
+
+## Adding enzyme ontogenies
+The protein expression database of PK-Sim includes ontogeny information for some proteins (see (OSPS manual)[https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#settings-in-the-protein-expression-tab]). For a protein molecule present in the simulation, it is possible to add the ontogeny information on one of the pre-defined proteins. For example, it is possible to set the ontogeny of the protein `MyProtein` to the value of the ontogeny of a CYP3A4 enzyme for the specified individual. The list of supported ontogenies is stored in the `StandardOntogeny`-list.
```

Comment:
Well, technically "protein expression database" is wrong, because this information is stored in PK-Sim database (and that's why only few proteins are supported here and not thousands from the expression-DB).
But I guess **for the docu** it does not really matter.
Otherwise: those are exactly the proteins described in https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/blob/master/PK-Sim%20Ontogeny%20Database%20Version%207.3.pdf

Reply:
Was not aware of that pdf - will link it in the vignette.